lua/syssp.c

```diff
@@ -228,8 +234,23 @@ static int db_comdb_verify(Lua L) {
         } else if (strcmp(m, "blobs") == 0) {
             mode = VERIFY_BLOBS;
             logmsg(LOGMSG_INFO, "Verify ONLY blobs for table %s\n", tblname);
-        } else
-            logmsg(LOGMSG_INFO, "Verify table %s\n", tblname);
+        } else if (strcmp(m, "serial") == 0) {
+            mode = VERIFY_SERIAL;
+            logmsg(LOGMSG_INFO, "Verify in serial mode table %s\n", tblname);
+        } else if (strcmp(m, "verbose") == 0) {
```

Comment:
May want to read all the args and apply flags?  Otherwise this allows "verbose", "verbose" for example.

Reply:
Yes, this does allow "verbose", "verbose", however i wanted to be able to omit the second parameter (ie assume some default, which for now is "serial") thus the best think i could think of is to allow the second param to be "verbose". What did you have in mind exactly?